projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -1,24 +1,389 @@
+    }
+  }
+
+  public static void saveTrips(Path csvPath, String path) {
+      try(FileOutputStream output = new FileOutputStream(path)){
+        CSVParser parser = CSVParser.parse(csvPath, StandardCharsets.UTF_8, CSVFormat.DEFAULT
+          .withHeader(
+            "route_id",
+            "service_id",
+            "trip_id",
+            "trip_headsign",
+            "direction_id",
+            "shape_id"));
+        boolean isHeader = true; 
+        for (CSVRecord record : parser) {
+          if(isHeader){
+            isHeader = false;
+            continue;
+          }
+        Trip.Builder trip = Trip.newBuilder();
+        trip.setRouteId(Integer.parseInt(record.get("route_id")))
+            .setServiceId(Integer.parseInt(record.get("service_id")))
+            .setTripId(record.get("trip_id"))
+            .setTripHeadsign(record.get("trip_headsign"))
+            .setDirectionId(Integer.parseInt(record.get("direction_id")))
+            .setShapeId(record.get("shape_id"))
+            .build()
+            .writeDelimitedTo(output);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static void main(String[] args) throws IOException {
+    // Paths to the CSVs files
+    Path agencyTxt = Paths.get(args[0]);
+    Path calendarTxt = Paths.get(args[1]);
+    Path fareAttributesTxt = Paths.get(args[2]);
+    Path fareRulesTxt = Paths.get(args[3]);
+    Path routesTxt = Paths.get(args[4]);
+    Path shapesTxt = Paths.get(args[5]); 
+    Path stopTimesTxt = Paths.get(args[6]); 
+    Path stopsTxt = Paths.get(args[7]);
+    Path translationsTxt = Paths.get(args[8]);  //NOTICE: problem with CSVParser
+    Path tripsTxt = Paths.get(args[9]);
+
+    //Paths to the output files
+    String agencyOutputPath = args[args.length - 10];
+    String calendarOutputPath = args[args.length - 9];
+    String fareAttributesOutputPath = args[args.length - 8];
+    String fareRulesOutputPath = args[args.length - 7];
+    String routesOutputPath = args[args.length - 6];
+    String shapesOutputPath = args[args.length - 5]; 
+    String stopTimesOutputPath = args[args.length - 4]; 
+    String stopsOutputPath = args[args.length - 3];
+    String translationsOutputPath = args[args.length - 2];  //NOTICE: problem with CSVParser
+    String tripsOutputPath = args[args.length - 1];
+
+     saveAgency(agencyTxt, agencyOutputPath);
+     saveCalendar(calendarTxt, calendarOutputPath);
+     saveFareAttributes(fareAttributesTxt, fareAttributesOutputPath);
+     saveFareRules(fareRulesTxt, fareRulesOutputPath);
+     saveRoutes(routesTxt, routesOutputPath);
+     saveShapes(shapesTxt, shapesOutputPath);
+     saveStopTimes(stopTimesTxt, stopTimesOutputPath);
+     saveStops(stopsTxt, stopsOutputPath);
+     saveTranslations(translationsTxt, translationsOutputPath); //NOTICE: problem with CSVParser
+     saveTrips(tripsTxt, tripsOutputPath);
+     
+    //TODO: Remove this line after fix the saveTranslations

```

Comment:
Do we still have a problem here?
Also 2 notes of:
`//NOTICE: problem with CSVParser`